scripts/UPDATE.sh

```diff
@@ -35,19 +35,22 @@ trap 'echo An error occurred in $0 at line $LINENO. Current working-dir: $PWD' E
 SB_TAG='default'
 num_parallel=2
 update_remote=0
-while getopts hrt:j: option
+apt_install=0
+while getopts hrst:j: option
  do
  case "${option}"
  in
   r) update_remote=1;;
+  s) apt_install=1;;
   t) SB_TAG=$OPTARG;;
   j) num_parallel=$OPTARG;;
   h)
-   echo "Usage: $0 [-t tag] [-j n] [-r] "
+   echo "Usage: $0 [-t tag] [-j n] [-r] [-s]"
    echo "Use the tag option to checkout a specific version of the SIRF-SuperBuild."
    echo "   Otherwise the most recent release will be used."
    echo "Use the -j option to change the number of parallel builds from the default ${num_parallel}"
    echo "Use the -r option to reset your git remotes to default SyneRBI sources."
+   echo "Use the -s option to run apt install for all prerequisites (use with caution)."
```

Comment:
can you find a way to generalise this beyond `apt` in a few words? (as it's doing more)